src/renderer/component/wunderbar/internal/autocomplete.jsx

```diff
@@ -45,129 +44,129 @@ function getScrollOffset() {
-     * { top, left, minWidth } which are the coordinates of the top-left corner
-     * and the width of the dropdown menu.
-     */
-    renderMenu: PropTypes.func,
-    /**
-     * Styles that are applied to the dropdown menu in the default `renderMenu`
-     * implementation. If you override `renderMenu` and you want to use
-     * `menuStyle` you must manually apply them (`this.props.menuStyle`).
-     */
-    menuStyle: PropTypes.object,
-    /**
-     * Arguments: `props: Object`
-     *
-     * Invoked to generate the input element. The `props` argument is the result
-     * of merging `props.inputProps` with a selection of props that are required
-     * both for functionality and accessibility. At the very least you need to
-     * apply `props.ref` and all `props.on<event>` event handlers. Failing to do
-     * this will cause `Autocomplete` to behave unexpectedly.
-     */
-    renderInput: PropTypes.func,
-    /**
-     * Props passed to `props.renderInput`. By default these props will be
-     * applied to the `<input />` element rendered by `Autocomplete`, unless you
-     * have specified a custom value for `props.renderInput`. Any properties
-     * supported by `HTMLInputElement` can be specified, apart from the
-     * following which are set by `Autocomplete`: value, autoComplete, role,
-     * aria-autocomplete. `inputProps` is commonly used for (but not limited to)
-     * placeholder, event handlers (onFocus, onBlur, etc.), autoFocus, etc..
-     */
-    inputProps: PropTypes.object,
-    /**
-     * Props that are applied to the element which wraps the `<input />` and
-     * dropdown menu elements rendered by `Autocomplete`.
-     */
-    wrapperProps: PropTypes.object,
-    /**
-     * This is a shorthand for `wrapperProps={{ style: <your styles> }}`.
-     * Note that `wrapperStyle` is applied before `wrapperProps`, so the latter
-     * will win if it contains a `style` entry.
-     */
-    wrapperStyle: PropTypes.object,
-    /**
-     * Whether or not to automatically highlight the top match in the dropdown
-     * menu.
-     */
-    autoHighlight: PropTypes.bool,
-    /**
-     * Whether or not to automatically select the highlighted item when the
-     * `<input>` loses focus.
-     */
-    selectOnBlur: PropTypes.bool,
-    /**
-     * Arguments: `isOpen: Boolean`
-     *
-     * Invoked every time the dropdown menu's visibility changes (i.e. every
-     * time it is displayed/hidden).
-     */
-    onMenuVisibilityChange: PropTypes.func,
-    /**
-     * Used to override the internal logic which displays/hides the dropdown
-     * menu. This is useful if you want to force a certain state based on your
-     * UX/business logic. Use it together with `onMenuVisibilityChange` for
-     * fine-grained control over the dropdown menu dynamics.
-     */
-    open: PropTypes.bool,
-    debug: PropTypes.bool,
-  };
+  // static propTypes = {
+  //   /**
```

Comment:
Why is this commented out?

Reply:
I wanted to get rid of `prop-types` since we aren't using it. This is from a third party library but we needed to make a change to the component so I just copied it in.

I still wanted to keep them for documentation.